src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/Services/AccountService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Text.Json;
+
+using Umbraco.Forms.Integrations.Crm.ActiveCampaign.Models.Dtos;
+
+namespace Umbraco.Forms.Integrations.Crm.ActiveCampaign.Services
+{
+    public class AccountService : IAccountService
+    {
+        private readonly IHttpClientFactory _httpClientFactory;
+
+        public AccountService(IHttpClientFactory httpClientFactory)
+        {
+            _httpClientFactory = httpClientFactory;
+        }
+
+        public async Task<AccountCollectionResponseDto> Get()
+        {
+            var client = _httpClientFactory.CreateClient(Constants.HttpClient);
+
+            var response = await client.SendAsync(
```

Comment:
Doesn't really matter, but you could use `GetAsync` and `PostAsync` instead of `SendAsync` if you preferred.... might simplify the code a little bit.